resources/views/entities/pages/inventory/_grid.blade.php

```diff
@@ -29,11 +28,18 @@ class="btn2 btn-default btn-sm"
                         <a href="#" class="rounded hidden link link-accent bg-box">
                             <x-icon class="fa-solid fa-times" />
                         </a>
+                        <td class="text-right">
+                            <x-button.delete-confirm size="sm" target="#delete-position-{{ \Illuminate\Support\Str::slug($position) }}" iconOnly=true />
+                            {!! Form::open(['method' => 'DELETE',
+                                'route' => ['entities.inventory.delete.section', $campaign, $entity, 'position' => $position],
```

Comment:
Due to special characters (and having very ugly urls), pass the inventory_item instead